server/services/reportedAdjudicationsService.ts

```diff
@@ -485,7 +486,11 @@ export default class ReportedAdjudicationsService {
     return newDraftAdjudicationData.draftAdjudication.id
   }
 
-  async getPrisonerReport(user: User, adjudication: DraftAdjudication & ReportedAdjudication): Promise<PrisonerReport> {
+  async getPrisonerReport(
+    user: User,
+    adjudication: DraftAdjudication & ReportedAdjudication,
+    draftRequired?: boolean
```

Comment:
In `prisonerReportPage.prisonerReportDetails` when `draftRequired=true` then we call `reportedAdjudicationsService.createDraftFromCompleteAdjudication`, thought made sense to reuse that flag to decide if we show the _Change_ link as we only want to display it when editing a draft